app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md

```diff
@@ -86,38 +86,38 @@ The main visa route for non-UK citizens wishing to teach in England is the skill
 
 You’ll be able to apply for your skilled worker visa if:
 
-* you have a job offer from a school that is a Home Office licensed visa sponsor 
+* you have a job offer from a school that is a Home Office licensed visa sponsor
 * you have a certificate of sponsorship from this school, confirming they have offered you a job and will sponsor you
 * [you meet certain standards in speaking, reading, writing and understanding English](https://www.gov.uk/skilled-worker-visa/knowledge-of-english)
 * your salary meets the requirement for a skilled worker visa
 
 [Learn more about applying for a teaching job in England](/non-uk-teachers/teach-in-england-if-you-trained-overseas).
 
 ### Salary requirement for a skilled worker visa
- 
+
 The salary you must be paid to get a visa varies according to  whether you are employed as a ‘qualified’ or ‘unqualified’ teacher. This depends on your level of experience and qualifications – your employing school can explain whether the role you are applying for is qualified or unqualified. The salary requirement also varies depending on the region you are applying to work in.
 
 If you are employed as a qualified teacher and work in:
 
-* inner London, you must earn £36,745
-* outer London, you must earn £34,514
-* London fringe, you must earn £31,350
-* the rest of England, you must earn £30,000
+* inner London, you must earn $salaries_skilledworkervisateacherminfulltime_innerlondon$
+* outer London, you must earn $salaries_skilledworkervisateacherminfulltime_outerlondon$
+* London fringe, you must earn $salaries_skilledworkervisateacherminfulltime_londonfringe$
+* the rest of England, you must earn $salaries_skilledworkervisateacherminfulltime_restofengland$
 
 If you are employed as an unqualified teacher and work in:
 
-* inner London, you must earn £25,831
-* outer London, you must earn £24,415
-* London fringe, you must earn £23,200
-* the rest of England, you must earn £23,200
+* inner London, you must earn $salaries_skilledworkervisatraineeminfulltime_innerlondon$
```

Comment:
Question of whether we need to create a new variable for these given this is the 'non-uk-teachers' page. Do we want to have:
`$salaries_skilledworkervisa**unqualified**teacherminfulltime_restofengland$`

Reply:
Move anything salary related to non-uk config file - these relate to salary earnings

Reply:
$salaries_skilledworkertraineefulltimemin_innerlondon$ change trainees to teachers and also add unqualified to the variable